### PR TITLE
Changed e2e functional tests to configure settings for AB#15003

### PR DESCRIPTION
**Pull request by rayking29**
# Fixes or Implements [AB#15003](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/15003)

## Testing

- [ ] Unit Tests Updated
- [x] Functional Tests Updated
- [ ] Not Required

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by ytqsl — APPROVED** [review]
I think this is the last pass, so @rayking29 you can probably remove enableModules and the 2nd param of configureSettings

File: Testing/functional/tests/cypress/integration/e2e/timeline/comments.js [hunk 1]
```diff
@@ -2,30 +2,47 @@ const { AuthMethod } = require("../../../support/constants");
...
-                },
+        cy.configureSettings({
+            timeline: {
+                comment: true,
             },
-            "Laboratory"
-        );
+            datasets: [
+                {
+                    name: "clinicalDocument",
```
Comment (BrianMaki) [inline Testing/functional/tests/cypress/integration/e2e/timeline/comments.js:41]: why the change to 'clinicalDocument' from 'laboratory'?

**Review by ytqsl — APPROVED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Review by BrianMaki — APPROVED** [review]

File: Testing/functional/tests/cypress/integration/e2e/timeline/comments.js (hunk 1 above)
Comment (rayking29) [reply Testing/functional/tests/cypress/integration/e2e/timeline/comments.js]: Fixed

File: Testing/functional/tests/cypress/integration/e2e/covid19/publicCovidTest.js [hunk 2]
```diff
@@ -107,7 +107,19 @@ function checkResult(
 
 describe("Public COVID-19 Test Results", () => {
     beforeEach(() => {
-        cy.enableModules([laboratoryModule, publicLaboratoryResultModule]);
```
[the remark this answers is not in the document]
Comment (rayking29) [reply Testing/functional/tests/cypress/integration/e2e/covid19/publicCovidTest.js:110]: Removed.

File: Testing/functional/tests/cypress/integration/e2e/user/quickLinks.js [hunk 3]
```diff
@@ -199,16 +217,38 @@ describe("Quick Links", () => {
...
     beforeEach(() => {
-        cy.enableModules([
-            laboratoryModule,
-            encounterModule,
-            immunizationModule,
-            laboratoryModule,
-            allLaboratoryModule,
-            medicationModule,
-            medicationRequestModule,
-            noteModule,
```
[the remark this answers is not in the document]
Comment (rayking29) [reply Testing/functional/tests/cypress/integration/e2e/user/quickLinks.js:210]: Removed the ones I could, some are used for other purposes.

File: Testing/functional/tests/cypress/integration/ui/covid19/publicVaccineCard.js [hunk 4]
```diff
@@ -217,16 +217,19 @@ describe("Public Vaccine Card Form", () => {
 
 describe("Public Vaccine Card Downloads", () => {
     beforeEach(() => {
-        cy.configureSettings(
```
Comment (BrianMaki) [inline Testing/functional/tests/cypress/integration/ui/covid19/publicVaccineCard.js:220]: Why are there ui changes?  The feature branch is already changed.

https://github.com/bcgov/healthgateway/blob/features/13691/Testing/functional/tests/cypress/integration/ui/pages/publicRoute.js
